students/Snayxts/module-2-html/exercises/zdk09/index.html

```diff
@@ -0,0 +1,120 @@
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M37.4822 28.4545C36.784 30.2698 36.4349 32.473 36.4349 35.064C36.4349 39.6875 37.5908 43.1086 39.9026 45.3273C42.2298 47.5459 45.4182 48.6553 49.4677 48.6553C51.4226 48.6553 53.2378 48.407 54.9135 47.9105C56.6046 47.4141 58.0631 46.615 59.2888 45.5134C60.5145 44.3964 61.4764 42.9302 62.1746 41.1149C62.8883 39.2996 63.2451 37.0887 63.2451 34.4822C63.2451 29.8431 62.0892 26.4143 59.7775 24.1956C57.4657 21.977 54.2774 20.8676 50.2124 20.8676C48.273 20.8676 46.4577 21.1236 44.7666 21.6356C43.0754 22.1321 41.617 22.9389 40.3913 24.056C39.1656 25.1731 38.1959 26.6393 37.4822 28.4545ZM53.6427 36.9132C55.2885 33.3838 56.1674 30.7464 56.2794 29.0011C56.4054 27.2624 55.695 26.0324 54.1483 25.3111C52.8687 24.7144 51.594 24.9332 50.3242 25.9674C49.0751 26.9942 47.6539 29.216 46.0605 32.6329C44.4147 36.1624 43.5288 38.7964 43.4028 40.5352C43.2908 42.2805 44.0082 43.5138 45.555 44.235C46.8346 44.8317 48.1022 44.6097 49.3579 43.5689C50.6277 42.5347 52.0559 40.3161 53.6427 36.9132Z" fill="black"/>
+        </svg>
+
+        <h1>Company - Better Together</h1>
+
+        <nav>
+            <a href="#aboutus">About</a><br>
+            <a href="#productss">Products</a><br>
+            <a href="#contactt">Get in touch</a><br>
+            <a href="#inf">Additional information</a><br>
+        </nav>
+    </header>
+    <br>
+    
+    <main>
+        <header>
+            <h2>Boring can be fun</h2>
+        </header>
+
+        <section>
+            <p>Tech virtual drone online browser platform through in a system. But stream software offline. Professor install angel sector anywhere create at components smart. Document fab developers encryption smartphone powered, bespoke blockstack edit atoms. Companies a storage adopters. Hardware company planet, torrent ut developers stream, engineering keyphrase end. Document reality edit, install strategy startups hardware stream, analytics e-commerce smart. Privacy news data policies analytics documents.</p>
+            <p>Now digital designs id anywhere atoms. Now strategy startups documents designs. Venture crypto adopters niche. Video algorithm system ultra-private policies engineering. Users takedowns. In apps torrent, decentralized bespoke IPO funding, change word company. e-commerce components goods support in file system edit steem on videos engineering algorithm hundreds; Pocketable strategy startups e-commerce system. Document 1,000 how torrent, non security mi sell crypto. Despite clever designers how, word steem ability analytics.</p>
+        </section>
+
+    
+        <article>

```

Comment:
Ovaj segment stranice ne bi bio dobar članak jer je sadržaj u njemu veoma kontekstualan. "How can we help" i "What we do and why we do it" bi bolje poslužile kao sekcije unutar glavnoj sadržaja stranice.